src/systray.c

```
@@ -328,17 +328,24 @@ void systray_about_dialog()
 void systray_click_cb(GtkStatusIcon* icon, GdkEventButton* ev, gpointer userdata)
 {
     menu_infos_t* mis = userdata;
+    menu_info_t* mi = &mis->menu_info[(ev->state & GDK_CONTROL_MASK) ? MENU_SOURCE : MENU_SINK];
+    menu_info_item_t* mii = menu_info_item_get_by_name(mi, mi->default_name);
 
     switch(ev->button)
     {
         case 1:
+            {
+                if((ev->state & GDK_MOD1_MASK) && mii)
```

Is the check for mii != NULL necessary? I don't think the function should ever return NULL. If it ever does, let it crash so we can fix the root of the problem.